README.md

```diff
@@ -1,5 +1,80 @@
+
+## Prerequisites
+
+You need a Dynatrace environment to export your spans to.
+
+We currently only offer support and documentation for using this package on Google Cloud Functions (GCF). For GCF-specific prerequisites, please refer to [Prerequisites](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/opentelemetry-integration/opentelemetry-on-gcf-go#prerequisites).
+
+## Getting started
+
+This README file just contains a very brief quickstart code sample. The main documentation of this package can be found in the [Dynatrace Documentation](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/opentelemetry-integration/opentelemetry-on-gcf-go).
+
+Follow [the instructions](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/opentelemetry-integration/opentelemetry-on-gcf#choose-config-method) on how to set up the Dynatrace configuration for your project by using a config file or environment variables. An error will be returned during the TracerProvider instantiation if the config cannot be found.
+
+1. Import the package into your Go project:
+    ```shell
+    go get github.com/dynatrace-oss/opentelemetry-exporter-go/core
+    ```
+2. Create a Dynatrace TracerProvider and TextMapPropagator:
+    ```go
+    import (
+        "go.opentelemetry.io/otel"
+        dtTrace "github.com/dynatrace-oss/opentelemetry-exporter-go/core/trace"
+    )
+
+    // call this once to initialize all required tracing components in your program
+    func initializeTracing() error {
+        // create a TracerProvider
+        tracerProvider, err := dtTrace.NewTracerProvider()
+        if err != nil {
+            // handle error as needed
+            return err
+        }
+        otel.SetTracerProvider(tracerProvider)
+
+        // create a TextMapPropagator
+        propagator, err := dtTrace.NewTextMapPropagator()
+        if err != nil {
+            // handle error as needed
+            return err
+        }
+        otel.SetTextMapPropagator(prop)
+    }
+    ```
+3. Use the OpenTelemetry API as you normally would to create spans:
+    ```go
+    // otel.GetTracerProvider() will now return the DtTracerProvider that was created previously, it is safe to type-assert.
+    tracerProvider := otel.GetTracerProvider().(*dtTrace.DtTracerProvider)
+
+    // create a tracer and spans
+    tracer := tracerProvider.Tracer("example tracer")
+    ctx, span := tracer.Start(context.Background(), "example span")
+    // do something between starting and ending the span
+    span.End()
+
+    // ensure the DtTracerProvider is properly shut down when you are done creating spans
+    // spans will be flushed, and any spans created after this call will no longer be processed or exported 
+    tracerProvider.Shutdown(ctx)
```

Comment:
I don't think it makes sense to call shutdown in the context of the ended span. I also think we should be using a defer statement right after we assign the tracerProvider variable.

```suggestion
    tracerProvider := otel.GetTracerProvider().(*dtTrace.DtTracerProvider)
   
    // ensure the DtTracerProvider is properly shut down when you don't need it anymore.
    // spans will be flushed, and after the deferred call executes, spans will no longer be processed or exported (even if started before)
    defer tracerProvider.Shutdown(context.Background())

    // create a tracer and spans
    tracer := tracerProvider.Tracer("example tracer")
    ctx, span := tracer.Start(context.Background(), "example span")
    // do something between starting and ending the span
    span.End()
```